.github/workflows/rs-drive-abci.yml

```diff
@@ -33,15 +33,16 @@ jobs:
     with:
       package: "drive-abci"
 
-  docker:
-    name: Docker
-    runs-on: self-hosted
-    timeout-minutes: 180  # ARM build can take forever
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/docker
-        with:
-          dockerfile: ./Dockerfile
-          image: dashpay/drive-abci
-          target: drive-abci
-          push: false
+# TODO: Remove?
```

Comment:
What was the purpose of this? Just to check if Docker build works?

Reply:
Yes. There is no need for that since we build Drive in test suite workflow.